core/controllers/admin.py

```diff
@@ -94,6 +94,8 @@ class AdminHandler(base.BaseHandler):
                         'update_feature_flag_rules'
                     ]
                 },
+                # TODO(#13331): Remove default_value once clear_search_index
+                # is fixed.
```

Comment:
Please add additional information to the TODO, it is too vague, what does clear_search_index is fixed mean?